templates/csi.yaml.j2

```diff
@@ -405,13 +405,20 @@ spec:
               mountPath: "/dev"
             - name: varlog
               mountPath: "/var/log/gluster"
+            - name: mountpath
+              mountPath: /mnt
         - name: kadalu-logging
           image: busybox
           command: ["/bin/sh"]
           args: ["-c", "while true; do logcnt=$(/bin/ls /var/log/gluster/ | wc -l); if [ ${logcnt} -gt 0 ]; then break; fi; sleep 5; done; tail -f /var/log/gluster/*.log"]
           volumeMounts:
             - name: varlog
               mountPath: "/var/log/gluster"
+        - name: kadalu-quota-crawler
```

Comment:
This can be removed if we move quota crawler to csi container itself.(Ref comment above)